di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/domain/Credential.java

```diff
@@ -7,21 +7,18 @@ public class Credential {
     private final Map<String, Object> evidence;
     private final String userId;
     private final String clientId;
-    private final Long exp;
```

Comment:
It does, but it can be removed as CRIs no longer set `exp`, or at least shouldn't. And we don't use it for anything - we use `nbf` for doing expiry stuff. So this PR pulls out the expiry stuff to keep it simpler overall.

The next step would be to remove all the stuff in the e2e tests that interacts with the expiry stuff in the CRI stub. And then finally remove those options from the CRI stub frontend.

Reply:
Ok, good to know - I guess the cleanup is in a separate ticket?

Reply:
It will be...